src/components/AuthStoreSetter.ts

```diff
@@ -1,21 +1,26 @@
 import _ from 'lodash/fp';
 import { useEffect } from 'react';
 import { AuthContextProps, useAuth } from 'react-oidc-context';
-import { loadAuthToken, OidcUser, processUser } from 'src/libs/auth';
+import { doUserLoaded, loadAuthToken, OidcUser } from 'src/libs/auth';
 import { useOnMount } from 'src/libs/react-utils';
-import { authStore } from 'src/libs/state';
+import { oidcStore } from 'src/libs/state';
 
 function AuthStoreSetter(): null {
+  // When the AuthContext changes, this component will reload
```

Comment:
I'm assuming this comment is referring to `useAuth`. If so, suggest clarifying "reload" to "rerender". This function will run again, but the component instance will be the same.

```suggestion
  // When the AuthContext changes, this component will rerender
```